LayoutTests/imported/w3c/web-platform-tests/css/css-backgrounds/inner-border-non-renderable-ref.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style>
+        .clipping {
+            margin: 10px;
+            width: 300px;
+            height: 200px;
+            overflow: hidden;
+            border: 30px solid green;
+            border-top-color: gold;
+            border-top-right-radius: 150px 267px;
+            /* border-bottom-left-radius: 150px 267px; */
```

Comment:
Is this commented out code useful? This whole file should probably match the -expected.html

Reply:
Fixed.